README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# hi! welcome to my repo. 
+## my name is sanaa imami and i'm a rising senior at the lab schools. 
+i intend to double major in **psychology** and **middle eastern studies**. 
```

Comment:
I like how you bolded your majors, so that they stand out as an important part of the text